### PR TITLE
[Don't Need Review] Implement connection Dialog and View

### PR DESCRIPTION
**Pull request by codeimpl**
This patch connects QuickMenuDialog and QuickMenuView. And it adds
onDismissed api to Delegate for update viewModel when dialog is
dismissed without request from controller.
    
Refs: #931

### CODE REVIEW

File: src/org/triple/banana/quick_menu/QuickMenuViewImpl.java [hunk 1]
```diff
@@ -5,18 +5,54 @@
...
 import androidx.annotation.NonNull;
 import androidx.annotation.Nullable;
 
+import org.triple.banana.R;
+
 class QuickMenuViewImpl implements QuickMenuView {
     private @Nullable Delegate mDelegate;
+    private @Nullable QuickMenuDialog mQuickMenuDialog;
+
+    public QuickMenuViewImpl(@NonNull Context context) {
```
Comment (jdragonbae) [inline src/org/triple/banana/quick_menu/QuickMenuViewImpl.java:21]: Can we change `public` to `package`?

File: src/org/triple/banana/quick_menu/QuickMenuFactoryImpl.java [hunk 2]
```diff
@@ -5,14 +5,16 @@
 
 package org.triple.banana.quick_menu;
 
+import android.content.Context;
+
 class QuickMenuFactoryImpl implements QuickMenuFactory {
     @Override
-    public QuickMenuController create() {
-        QuickMenuViewImpl view = new QuickMenuViewImpl();
+    public QuickMenuController create(Context context) {
```
Comment (jdragonbae) [inline src/org/triple/banana/quick_menu/QuickMenuFactoryImpl.java:12]: `@NonNull` Context context

File: src/org/triple/banana/quick_menu/QuickMenuFactory.java [hunk 3]
```diff
@@ -5,6 +5,8 @@
 
 package org.triple.banana.quick_menu;
 
+import android.content.Context;
+
 interface QuickMenuFactory {
-    QuickMenuController create();
+    QuickMenuController create(Context context);
```
Comment (jdragonbae) [inline src/org/triple/banana/quick_menu/QuickMenuFactory.java:11]: `@NonNull` QuickMenuController create(`@NonNull` Context context)

File: src/org/triple/banana/quick_menu/QuickMenuViewImpl.java (hunk 1 above)
Comment (codeimpl) [reply src/org/triple/banana/quick_menu/QuickMenuViewImpl.java]: done. thanks.

File: src/org/triple/banana/quick_menu/QuickMenuFactoryImpl.java (hunk 2 above)
Comment (codeimpl) [reply src/org/triple/banana/quick_menu/QuickMenuFactoryImpl.java]: done.

File: src/org/triple/banana/quick_menu/QuickMenuFactory.java (hunk 3 above)
Comment (codeimpl) [reply src/org/triple/banana/quick_menu/QuickMenuFactory.java]: done! :+1:

File: src/org/triple/banana/quick_menu/QuickMenuViewImpl.java (hunk 1 above)
Comment (codeimpl) [reply src/org/triple/banana/quick_menu/QuickMenuViewImpl.java]: done! :+1:

File: src/org/triple/banana/quick_menu/QuickMenuFactoryImpl.java (hunk 2 above)
Comment (codeimpl) [reply src/org/triple/banana/quick_menu/QuickMenuFactoryImpl.java]: done!

File: src/org/triple/banana/quick_menu/QuickMenuFactory.java (hunk 3 above)
Comment (codeimpl) [reply src/org/triple/banana/quick_menu/QuickMenuFactory.java]: done :)

**Pull request closed without merging**